testdata/early_return.go

```diff
@@ -5,28 +5,28 @@ package fixtures
 import "os"
 
 func earlyRet() bool {
-	if cond { //   MATCH /if c { ... } else { ... return } can be simplified to if !c { ... return } .../
+	if cond { //   MATCH /if c { ... } else { return } can be simplified to if !c { return } .../
```

Comment:
Let's keep the old failure message.

```suggestion
	if cond { //   MATCH /if c { ... } else { ... return } can be simplified to if !c { ... return } .../
```

Reply:
Out of interest, is that an aesthetic preference or to avoid breaking things that might be depending on an exact match on the text?

Reply:
To clarify my question: the change being reflected in the test here is not that the ellipses have been removed. Rather they are just conditional on there being _something_ to elide. In this particular test case the `else` block consists of a single `return` statement and therefore shows as `{ return }`. Had there been some other statements preceding the return, it would be indicated by `{ ... return }`, with the `...` standing in for those other statements.

Happy to revert if that is indeed understood, just wanted to make sure.

Reply:
> Out of interest, is that an aesthetic preference or to avoid breaking things that might be depending on an exact match on the text?

It's to avoid breaking things. Someone may exclude linter by text in golangci-lint by using the `exclude-rules.text`. See https://github.com/golangci/golangci-lint/blob/b7b77ca30e2e77fb55da8bd04faf4b2a7d8d76d8/.golangci.reference.yml#L3746